README.md

```diff
@@ -143,6 +143,15 @@ This section describes how to get started developing Chisel itself, including ho
 
 ### Compiling and Testing Chisel
 
+First, you need to clone and build the [firrtl](https://github.com/freechipsproject/firrtl) and [treadle](https://github.com/freechipsproject/treadle) projects:
```

Comment:
```suggestion
First, clone and build the master branch of [FIRRTL](https://github.com/freechipsproject/firrtl) and [Treadle](https://github.com/freechipsproject/treadle), since the master branch of Chisel may depend on breaking-edge features:
```